sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/AppDestination.kt

```diff
@@ -1,15 +1,19 @@
 package co.nimblehq.sample.compose.ui
 
 import androidx.navigation.*
+import co.nimblehq.sample.compose.model.UiModel
 
 const val KeyId = "id"
+const val ModelKey = "model"
```

Comment:
Should we put the `Key` as a prefix instead? At least it is consistent with `KeyId`
```suggestion
const val KeyModel = "model"
```